### PR TITLE
[DPTP-1433] Add the registry-replacer step in the autoconfigbrancher tool

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform @alvaroaleman 

follow-up https://github.com/openshift/release/pull/15700

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: cmd/autoconfigbrancher/main.go [hunk 1]
```diff
@@ -195,6 +195,22 @@ func main() {
 				"--prune=true",
 			},
 		},
+		{
+			command: "/usr/bin/registry-replacer",
```
Comment (alvaroaleman) [inline cmd/autoconfigbrancher/main.go:199]: This needs to run first, before the config brancher

Comment (droslean) [reply cmd/autoconfigbrancher/main.go]: updated

File: cmd/autoconfigbrancher/main.go [hunk 2]
```diff
@@ -137,6 +137,22 @@ func main() {
 		command   string
 		arguments []string
 	}{
+		{
+			command: "/usr/bin/registry-replacer",
+			arguments: []string{
+				"--github-token-path", "/etc/github/oauth",
+				"--github-endpoint", "http://ghproxy",
+				"--config-dir", "./ci-operator/config",
+				"--create-pr",
```
Comment (alvaroaleman) [inline cmd/autoconfigbrancher/main.go:146]: This needs to be `false` :P

Comment (droslean) [reply cmd/autoconfigbrancher/main.go:146]: Nice catch! Yea since we moved the job in autoconfigbrancher it doesn't need to create a pr. Thanks.

**Review by alvaroaleman — APPROVED** [review]

**Comment by droslean** [comment]
I will force merge this since the openshift/release#15700 merged first.

**Pull request merged**